File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    do
    {
        // Prompting user for height
        height = get_int("To draw a pyramid please enter a number between 1 and 8: ");
    }
    // Checking if user typed a positive integer and an integer between 0 and 8
    while (height < 1 || height > 8);
    
    // Print number of lines
    for (int lines = 0; lines < height; lines++)
    {
        // Spaces
        for (int space = height - lines; space > 1; space--)
        {
            printf(" ");
        }
        
        // Hashes
        for (int hash = 0; hash < lines + 1; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

# Prompting user for a positive integer and an integer between 0 and 8
while True:
    height = get_int("To draw a pyramid please enter a number between 1 and 8: ")
    if height > 0 and height < 9:
        break

# Prints number of lines
for lines in range(height):
    # Prints number of spaces
    for space in range(height - lines - 1):
        print(" ", end="")

    # Prints hashes
    for hashes in range(lines + 1):
        print("#", end="")

    # Prints new line
    print("")
